v3/fleet-local/control/flight-director@.service

```diff
@@ -39,6 +39,11 @@ ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
   -e FD_LOG_MARATHON_API_CALLS=`etcdctl get /FD/FD_LOG_MARATHON_API_CALLS` \
   -e FD_MARATHON_MASTER=`etcdctl get /FD/FD_MARATHON_MASTER` \
   -e FD_MESOS_MASTER=`etcdctl get /FD/FD_MESOS_MASTER` \
+  -e GITHUB_ALLOWED_TEAMS=`etcdctl get /FD/GITHUB_ALLOWED_TEAMS` \
```

Comment:
Pretty sure these were left out on purpose. They were only used for HUD (which is dead). Did anything break because they were removed?